platforms/Cross/plugins/JPEGReadWriter2Plugin/sqJPEGReadWriter2Plugin.c

```diff
@@ -54,9 +54,10 @@ primJPEGWriteImageonByteArrayformqualityprogressiveJPEGerrorMgrWriteScanlines(
 	error_ptr2 pjerr = (error_ptr2)jpegErrorMgr2Struct;
 
 	pcinfo->err = jpeg_std_error(&pjerr->pub);
+    pjerr->setjmp_buffer = (jmp_buf *) malloc(sizeof(jmp_buf));
```

Comment:
Hmm, I just noticed the difference of alignment, it seems like Xcode default set up was ruled by space lovers.
I'm committing with tabs.